src/pages/MainPage.css

```diff
@@ -0,0 +1,37 @@
+header {
```

Comment:
This would affect all the `header` tags. if you have   a `.Header` class, why not style that directly?